onchain/rollups/test/foundry/inputs/InputBox.t.sol

```diff
@@ -22,60 +24,82 @@ contract InputBoxTest is Test {
 
     function testAddLargeInput() public {
         address app = vm.addr(1);
-        uint256 max = CanonicalMachine.INPUT_PAYLOAD_MAX_SIZE;
+        uint256 max = _getMaxInputPayloadLength();
 
         _inputBox.addInput(app, new bytes(max));
 
+        uint256 largeLength = EvmAdvanceEncoder
+            .encode(address(this), 1, new bytes(max + 1))
```

Comment:
You can store the `new bytes(max + 1)` value in a variable called `payload`, since you'll be passing that to `addInput` later anyway.